Makefile

```diff
@@ -6,6 +6,7 @@ CARGOFLAGS =
 RONN = ronn
 PKG_CONFIG = pkg-config
 PREFIX = /usr
+BUILDTYPE = release
```

Comment:
```
BUILDTYPE ?= release
CARGOFLAGS += --$(BUILDTYPE)
```
will make the body cleaner and fix clippy being build in debug by default

Reply:
I'd prefer defaulting to `debug`, I think rust devs would expect that